src/features/presentation/components/carousel/carousel.component.html

```diff
@@ -38,6 +38,14 @@ <h3 class="h5 text-secondary fw-bold">{{ slides.titre }}</h3>
               <span class="visually-hidden">Accéder à DORA</span>
               <img class="my-3" height="100px" src="/assets/img/presentation/dora-logo.svg" alt="" />
             </a>
+            <a
+              *ngIf="slides.titre === 'Orientation'"
+              (click)="scrollToAnchor.emit('webinaire')"
+              class="rounded-0 me-4 text-secondary p-0"
+              style="cursor: pointer">
+              <i class="ri-arrow-down-line" style="font-weight: 100" aria-hidden="true"></i>
+              En savoir plus
+            </a>
```

Comment:
```suggestion
            <button
              *ngIf="slides.titre === 'Orientation'"
              (click)="scrollToAnchor.emit('webinaire')"
              class="me-4 p-0 btn btn-link text-decoration-none">
              <i class="ri-arrow-down-line" aria-hidden="true"></i>
              En savoir plus
            </button>
```
Il existe une classe bootstrap pour faire ça : `style="font-weight: 100"`